src/main/java/com/umc/networkingService/domain/test/controller/TestController.java

```diff
@@ -32,4 +34,55 @@ public String example() {
     public String ignore() {
         return "무시되는 API";
     }
+
+    private final TestService tempService;
+
+    @GetMapping("/test")
+    public BaseResponse<TestResponse.TempTestDTO> testAPI() {
+
+        return BaseResponse.onSuccess(TestResponse.TempTestDTO.builder().testString("This is Test!").build());
+    /*
+    {
+      "timestamp": "2024-01-12T17:19:02.0172718",
+      "code": "COMMON200",
+      "message": "요청에 성공하였습니다.",
+      "result": {
+        "testString": "This is Test!"
+      }
+    }
+     */
+    }
+
+    @GetMapping("/exception")
```

Comment:
테스트 api이긴 한데 주석을 다는 것보다 swagger에 api에 대해 약간의 설명으로 바꾸는게 좋을 거 같습니다.
```suggestion
    @Operation(summary = "예외처리 API", description = "flag 값에 따라 예외를 발생시키는 API입니다.")
    @Parameter(name = "flag", description = "1인 경우 예외처리", example = "1", required = true)
    @ApiResponses(value = {
            @ApiResponse(responseCode = "200", description = "성공"),
            @ApiResponse(responseCode = "TEMP4001", description = "예외 처리 테스트입니다.")
    })
```